### PR TITLE
WT-9086 Run flush_tier and checkpoint threads during populate for tiered

### PR DESCRIPTION
**Pull request by sueloverso**
storage.

### CODE REVIEW

**Comment by sueloverso** [comment]
@fedorova and @keitharnoldsmith  this is a clean diff of the minimum changes I need to successfully run flush_tier and checkpoint during the populate phase for wtperf. Please review. I have run it on the test configuration I have been using.

File: bench/wtperf/wtperf.c [hunk 1]
```diff
@@ -1335,8 +1338,12 @@ checkpoint_worker(void *arg)
             if (wtperf->stop)
                 break;
         }
-        /* If the workers are done, don't bother with a final call. */
-        if (wtperf->stop)
+        /*
+         * If the workers are done, don't bother with a final call unless the flush tier worker
+         * needs to a final checkpoint to complete. The checkpoint thread keeps running as long as
+         * there is a flush tier worker thread running.
```
Comment (keitharnoldsmith) [inline bench/wtperf/wtperf.c:1344]: This comment implies that we could keep doing checkpoints for a while if, for some reason, the flush thread keeps running.  In fact, the code will do one more checkpoint and then the checkpoint thread will leave this while loop and exit.  

I'm not sure if this is a faulty assumption in the code, or just an overly literal reading of the comment on my part.

Comment (keitharnoldsmith) [reply bench/wtperf/wtperf.c:1344]: That makes sense.  I'm good.

**Review by keitharnoldsmith — APPROVED** [review]
LGTM

**Pull request merged**